internal/vrf/ips.go

```diff
@@ -0,0 +1,66 @@
+package vrf
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+
+	"github.com/spf13/cobra"
+)
+
+func (c *Client) Ips() *cobra.Command {
+	var (
+		vrfID string
+		ipID  string
+	)
+
+	// createVRFCmd represents the creatVRF command
+	ipsVRFCmd := &cobra.Command{
+		Use:   "ips [-v <VrfID] [-p <ip_id>]",
```

Comment:
We should stick with the existing format for command-line flags:
1. `--vrf-id` matches the format we use for `--project-id`.
2. `--id/-i` instead of `--ipID/-p` since this is a command for working with IPs